anchore_engine/services/apiext/api/controllers/images.py

```diff
@@ -204,7 +201,25 @@ def make_response_vulnerability(vulnerability_type, vulnerability_data):
         "feed_group": "feed_namespace",
     }
     scan_result = vulnerability_data["cpe_report"]
+    # gather a map of package path->CVE IDs referred by vulndb entries, dedup logic copied from policy engine
```

Comment:
that fixed the output issue.

Reply:
This issue arose once the new logic was introduced as it thought the cve data was presented twice but instead of presenting some information, it prevented the information from surfacing.

Reply:
@zhill, the change was necessary because of the switch over to CVE IDs. Previous dedup logic collects nvd refs (CVEs) for each vulnerability and filters those from the final list. After the switch over to CVE identifier, the dedup logic filtered out the vulnerabilities that it shouldn't have. In addition, dedup was based purely on the identifier without any consideration for artifact. That was incorrect behaviour and the fix addresses that as well